src/scss/partials/_body.scss

```diff
@@ -0,0 +1,14 @@
+// $background-color-mix: #fdfcfb;
+// $background-color-mix-with: #e2d1c3;
+
+body {
+  font-family: $font-family;
+  background: $primary-color;
+}
+
+footer {
+  margin: 0 auto;
+  width: 100%;
+  max-width: 250px;
+  padding-top: 35px;
```

Comment:
`p-` classes for padding. Don't hard code padding values when a utility class can do it already.

Reply:
yes i will do the changes